### DISCUSSION TITLE
How to send a POST request to a Webhook?

### DISCUSSION THREAD

**Discussion by anthonylogarta** [Budibase/budibase#7526] [Help] [answered]
Hi, is it possible to send a POST request to a Webhook other than using either Zapier or Integromat/Make? I use n8n.io and wanted to see if this is possible.

I am looking at this old discussion thread - https://github.com/Budibase/budibase/discussions/2172. However, I no longer see an option to send a POST request.

Thanks.

**Comment by andz-bb** [comment, marked as answer]
Hey @anthonylogarta,

n8n.io integration is an open issue here: #4400 

I saw in that thread that someone had been able to use the Integromat webhook integration as a workaround to hit n8n. I've just tested it locally with the Zapier integration as well and it seems to hit my n8n OK.

<img width="1620" alt="Screenshot 2022-08-31 at 11 56 24" src="https://user-images.githubusercontent.com/110921612/187663546-743014ac-e783-4d53-99e5-795119b466d7.png">

<img width="1095" alt="Screenshot 2022-08-31 at 11 59 19" src="https://user-images.githubusercontent.com/110921612/187663725-9be6881a-4fd9-46c0-adf3-e12197347d67.png">

Hopefully this is useful for you as a workaround while we wait for full integration! If you have any further questions on this let me know and I'll be happy to help 😊

**Comment by anthonylogarta** [comment]
Thank you! I tested with the Integromat integration and it worked.
